Automattic/woocommerce-payments
# Loads the Sift tracking JS after onboarding, regardless of where the merchant has been dropped into.

Fixes https://github.com/Automattic/woocommerce-payments-server/issues/1428

Loads the Sift tracking JS after onboarding, regardless of where the merchant has been dropped into.

To test:
- Make sure you disable any and all adblockers / tracker blockers. That tripped me up for a while!
- Upload this branch of the plugin to a Jurassic Ninja site. Don't activate it.
- Make sure you're in Test Mode for WCPay. The easiest way is to install the WCPay Dev plugin (make sure no other overrides are enabled).
- Go to `WooCommerce -> Home` and click on the "Set up Payments" task item.
- Go through Jetpack Auth and Stripe's KYC as usual.
- You'll be dropped into the same page you started KYC from (WooCommerce dashboard page).
- Open the Network panel on your browser, search "sift", you'll see that a Sift-related JS has been enqueued.

I also went to the Sift Dashboard to see if the `$create_account` event now has browser-related data associated with it, but I couldn't see that event at all. @luizreis do you have any idea what could I have been doing wrong here?
